plugins/modules/remote_management/redfish/redfish_config.py

```diff
@@ -253,7 +269,8 @@
 CATEGORY_COMMANDS_ALL = {
     "Systems": ["SetBiosDefaultSettings", "SetBiosAttributes", "SetBootOrder",
                 "SetDefaultBootOrder"],
-    "Manager": ["SetNetworkProtocols", "SetManagerNic", "SetHostInterface"]
+    "Manager": ["SetNetworkProtocols", "SetManagerNic", "SetHostInterface"],
+    "SessionService": ["SetSessionService"]
```

Comment:
This way the next PR will have a smaller diff.
```suggestion
    "SessionService": ["SetSessionService"],
```